main.js

```diff
@@ -189,3 +189,26 @@ const popUp =document.querySelector('.pop_up-container')
 function openPop (){
   popUp.classList.add('open_pop-up');
 }
+
+// Form Validation
+
+const form =document.getElementById('contact_form');
+const error=document.getElementById('error');
+const email=document.getElementById('email');
+
+form.addEventListener('submit',(e)=>{
+     e.preventDefault();
+    validation();
+   });
```

Comment:
- [ ] Please 🙏🏻 fix your javascript logic because your form is not working even after entering valid data because of it a user can't send his/her query to you via email which is a bad user experience. Kindly fix the issue and even you can follow my below suggestion to fix the problem. 

```suggestion
form.addEventListener('submit',(e)=>{
    if(!validation()) {
      e.preventDefault()
      return false;
    }
});
```